README.md

```diff
@@ -848,7 +848,7 @@ ANSWERS.addComponent('FilterBox', {
   showCount: true,
   // Optional, execute a new search whenever a filter selection changes
   searchOnChange: false,
-  // Optional, show a reset button per filter group
+  // Optional, show a reset button per filter group, this will only display if searchOnChange is false
   resetFilter: false,
```

Comment:
We also want to make it clear that resetFilters behaves this way too.

Reply:
Oh true, I'm going to add it to all of the "reset*" config options to be clearer since it's possible that someone will only read the comment for the one they're trying to change.

Reply:
updated!